cypress-shared/support/location-availability/product.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  orange: {
```

Comment:
Move this to common/utils.js

export const PRODUCTS_LINK_MAPPING = {
orange: a[href*="/navel-oranges-grown-large-fresh-fruit/p"]'
coconut: 'a[href="/fresh-coconuts/p"]',
}